### DISCUSSION TITLE
Unbind and bind `<C-key>` keys doesnt work on vi key bindings

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [fish-shell/fish-shell#10551] [Q&A]
```fish
   set fish_greeting # Disable greeting
   set -g fish_key_bindings fish_vi_key_bindings
   bind --erase \ck
   bind --erase \cj
   bind \ck up-or-search
   bind \cj down-or-search
```
I am trying to set `<C-k>` `<C-j>` to `down/up-or-search` but is seems to not be
possible while also having `fish_vi_key_bindings` set, am i missing something?

**Comment by faho** [comment]
`bind` operates on "default" (what vi calls "normal") by default.

Use `--mode insert` to change bindings in insert mode (and similar for other modes).

There is also no need to `erase` bindings before setting them, you can just set them and it will override them.
